ta-sdk-spi/src/main/java/com/ibm/ta/sdk/spi/assess/RecommendationJson.java

```diff
@@ -77,7 +77,11 @@ public RecommendationJson(Recommendation recommendation, Environment environment
         // Build map of issues by category
         Map<String, List<Issue>> issuesMap = new LinkedHashMap<String, List<Issue>>();
         for (Issue issue : auIssues) {
-          String issueCat = issue.getCategory().getId();
+          IssueCategory iCat = issue.getCategory();
+          if (iCat == null) {
+            throw new TAException("No matching issue category found for issue with ID:" + issue.getId());
```

Comment:
IS TAException new? This should probably be SDKException
Whats the context in which this will happen?